app/Http/Controllers/Api/SubscriberController.php

```diff
@@ -17,6 +18,14 @@ class SubscriberController extends Controller
      */
     public function subscribe(Request $request)
     {
+        $validator = Validator::make($request->all(), [
```

Comment:
don't use this, please use form validation [https://laravel.com/docs/6.x/validation#form-request-validation](https://laravel.com/docs/6.x/validation#form-request-validation) to make controller cleaner

Reply:
Ok, I see

Reply:
I had made form request and implement it to controller, but I can't access my api because **This action is unauthorized**. even though, I had set true on authorize function.

`public function authorize()
    {
        return true;
    }`

Reply:
try to remove `guest` middleware from your route

Reply:
I had remove `guest` in the route like this

`Route::group(['namespace' => 'Api', 'prefix' => 'subscribe'], function () {
    Route::post('', 'SubscriberController@subscribe')->name('subscribe');
});`

but, it still return **_This action is unauthorized._**

Reply:
could you push your code ? because i just tried make form request and there is no problem

Reply:
Ok, otw push 😅🙈